hack/devtools/deploy-shared-env.sh

```diff
@@ -39,12 +39,17 @@ deploy_env_dev() {
         -n env-development \
         --template-file deploy/env-development.json \
         --parameters \
+            "agentPrefix=ARO-RHEL" \
```

Comment:
```suggestion
            "agentPrefix=ARO-RHEL-" \
```

(to correspond with below?)